packages/dev/core/src/Materials/meshDebugPluginMaterial.ts

```diff
@@ -285,19 +277,25 @@ export class MeshDebugPluginMaterial extends MaterialPluginBase {
     private static _pluginCount: number = 0;
 
     /**
-     * Index of this instance of the plugin.
-     * Based on pluginCount at time of instantiation.
-    */
-    @serialize()
-    private _pluginIndex: number;
-
+     * Color palette used for MATERIALIDS mode.
+     * Defaults to `defaultMaterialColors`
+     */
+    public static materialColors: Color3[] = defaultMaterialColors;

```

Comment:
nit: sorry, me again with the capitalization

```suggestion
    public static MaterialColors: Color3[] = defaultMaterialColors;
```